src/index.html

```diff
@@ -7,6 +7,498 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="nav">
+      <a
+        class="nav__link-logo"
+        href="#home"
+      >
+        <img
+          class="nav__logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav__container">
```

Comment:
```suggestion
      </a>
      
      <nav class="nav__container">
```
there should be a space between one level blocks

Reply:
fix code style according to the checklist all over the page